engine/taskpool-collector/src/main/kotlin/io/holunda/camunda/taskpool/sender/gateway/AxonCommandListGateway.kt

```diff
@@ -44,3 +47,26 @@ open class AxonCommandListGateway(
   }
 
 }
+
+/**
+ * Error handler, logging the error.
+ */
+open class LoggingTaskCommandErrorHandler(private val logger: Logger) : TaskCommandErrorHandler {
+
+  override fun apply(commandMessage: Any, commandResultMessage: CommandResultMessage<out Any?>) {
+    logger.error("SENDER-006: Sending command $commandMessage resulted in error ${commandResultMessage.exceptionResult()}")
```

Comment:
If we have an exception here, shouldn't we provide it to the logger so it prints the stacktrace?